### PR TITLE
Download zipped workflows

### PR DESCRIPTION
**Pull request by vedran-kasalica**
The feature requires the corresponding RESTful APE [PR](https://github.com/sanctuuary/restape/pull/51) to be approved.

### CODE REVIEW

**Comment by vedran-kasalica** [comment]
@kretep although you made the initial changes, I would like you to look at the additional changes before merging.

File: src/stores/DomainStore.ts [hunk 1]
```diff
@@ -26,7 +26,7 @@ export class DomainStore {
   }
 
   async fetchData() {
-    const response = await fetch('/api/domain?select=*,topic_of_research(id,unique_label)');
+    const response = await fetch(`/api/domain?select=*,topic_of_research(id,unique_label)`);
```
Comment (kretep) [inline src/stores/DomainStore.ts:29]: Not sure why this change is here

**Review by kretep — COMMENTED** [review]
The changes seem fine, but I'm getting a 404 when I try it out. Is the backend up-to-date?

File: src/stores/DomainStore.ts (hunk 1 above)
Comment (vedran-kasalica) [reply src/stores/DomainStore.ts:29]: I noticed that you used \` with fetch in most of the cases fetch(\`...\`), so I updated this one.

**Comment by vedran-kasalica** [comment]
> The changes seem fine, but I'm getting a 404 when I try it out. Is the backend up-to-date?

The backend is not approved yet, I noticed some security issues that I might fix before it is merged.

File: src/stores/DomainStore.ts (hunk 1 above)
Comment (kretep) [reply src/stores/DomainStore.ts:29]: In this case it made me suspect you might have forgotten to interpolate some parameters

Comment (vedran-kasalica) [reply src/stores/DomainStore.ts:29]: I see, I was not aware of the difference in the syntax, so I will revert the change.

Comment (vedran-kasalica) [reply src/stores/DomainStore.ts]: Done

**Review by kretep — APPROVED** [review]
